RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -84,17 +87,21 @@ void HLTPFDiJetCorrCheckerWithDiTau::produce(edm::StreamID iSId, edm::Event& iEv
           if (reco::deltaR2(taus[iTau1]->p4(), myPFJet2.p4()) < dRmin2_)
             continue;
 
-          for (unsigned int iTau2 = iTau1 + 1; iTau2 < taus.size(); iTau2++) {
-            if (taus[iTau1]->pt() < extraTauPtCut_ && taus[iTau2]->pt() < extraTauPtCut_)
-              continue;
-
-            if (reco::deltaR2(taus[iTau2]->p4(), myPFJet1.p4()) < dRmin2_)
-              continue;
-            if (reco::deltaR2(taus[iTau2]->p4(), myPFJet2.p4()) < dRmin2_)
-              continue;
-
-            correctComb = true;
-            break;
+          if (taus.size() == 1) { correctComb = true; }
```

Comment:
In the case of 1 tau, shouldn't the pT requirement on `iTau1` be considered? (right now it isn't)

Reply:
Yes, thanks for pointing this out. It should be considered and I just added it.